### PR TITLE
V1.12.1

### PR DESCRIPTION
**Pull request by mathbunnyru**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes: https://github.com/conda-forge/gtest-feedstock/pull/59
Closes: https://github.com/conda-forge/gtest-feedstock/pull/58

### CODE REVIEW

**Comment by mathbunnyru** [comment]
@conda-forge-admin, please rerender

File: recipe/bld.bat [hunk 1]
```diff
@@ -11,6 +11,9 @@ xcopy /S %GTEST_DIR%\googletest\include %LIBRARY_INC%
 xcopy /S %GTEST_DIR%\googlemock\include %LIBRARY_INC%
 set GMOCK_DIR=googlemock
 
+REM make this a bit more debuggable
+echo on
+
```
Comment (mathbunnyru) [inline recipe/bld.bat]: Is it ok to keep this or should I remove these lines?

**Comment by mathbunnyru** [comment]
I fixed windows build. I don't know why, but the travis CI was created here: https://github.com/conda-forge/gtest-feedstock/pull/59
I'm not sure it's needed and what I should do with it.

**Comment by mathbunnyru** [comment]
Oh, I guess Travis CI has some queue.
It seems that everything is fine 🎉

**Comment by mathbunnyru** [comment]
@SylvainCorlay @marcelotrevisani  @scopatz @wesm @xhochy
may I ask someone to take a look and merge, please?

**Comment by wolfv** [comment]
@conda-forge-admin, please rerender

**Comment by wolfv** [comment]
Looks good to me!

**Pull request merged**